server/jetstream_api.go

```diff
@@ -1812,8 +1812,12 @@ func (s *Server) jsStreamInfoRequest(sub *subscription, c *client, a *Account, s
 			// while the new members work through the election and catchup process.
 			// Double check for that instead of exiting here and being silent. e.g. nats stream update test --replicas=3
 			js.mu.RLock()
-			rg, ourID := sa.Group, cc.meta.ID()
-			bail := !rg.isMember(ourID)
+			rg, bail := sa.Group, true
```

Comment:
You set bail to true, but it is then set to `!rg.isMember(ourID)` just below, so that does not really do anything.

Reply:
Fair, fixed.